infrastructure/base/modules/database/main.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  postgres_secret_json = {
+    username = "postgres"
+    password = random_password.postgresql_admin_generator.result
+  }
```

Comment:
should we rely on the stock `postgres` user?
in practice, this should be equivalent to the current setup where we create an ad-hoc user, except perhaps that `postgres` may have additional permissions granted via `pg_hba` (on the `postgres` db, for example, though in practice I can successfully `\c postgres` as `marxan-api` user already, in a local db), and also we would forego (if that's an issue to start with - probably not) the separation of users and permissions across `apidb` and `geodb` when they are within the same physical PostgreSQL server.

tl;dr I think it's ok, just listing some things to cross check.

Reply:
ah right, this is the `base` layer - all good then.